<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Block Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>781</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="project_tab">
     <attribute name="title">
      <string>Проект</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="project_tree"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="template_tab">
     <attribute name="title">
      <string>Шаблоны</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListView" name="template_list"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="save_as_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_project">
    <property name="title">
     <string>TIA</string>
    </property>
    <addaction name="connect_action"/>
    <addaction name="sync_action"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_template">
    <property name="title">
     <string>Шаблоны</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_project"/>
   <addaction name="menu_template"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="connect_action">
   <property name="text">
    <string>Подлючиться к проекту</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_as_action">
   <property name="text">
    <string>Сохранить Как ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="sync_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Синхронизировать</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
